src/gui/mzroll/mainwindow.cpp

```diff
@@ -1731,6 +1731,56 @@ void MainWindow::open()
         Q_UNUSED(continueButton);
     }
 
+    int sampleCount = 0;
+    QStringList sampleList;
+    int projectCount  = 0;
+    QStringList projectList;
+    Q_FOREACH (QString filename, filelist) {
+        if (fileLoader->isEmdbProject(filename) 
+            || fileLoader->isMzrollDbProject(filename)
+            || fileLoader->isMzRollProject(filename)) {
+                projectCount++;
+                projectList << filename;
+            } else {
+                sampleCount++;
+                sampleList << filename;
+            }
+    }
+    filelist.clear();
+    if (sampleCount && !projectCount) {
+        filelist = sampleList;
+    } else if (!sampleCount && projectCount) {
+        if (projectCount > 1) {
+            filelist << projectList[0];
+        } else {
+            filelist = projectList;
+        }
+    } else {
+        QMessageBox msgBox;
+        msgBox.setText("It seems you are trying to load sample files and "
+                       "project files on one instance.\n\n"
```

Comment:
"It seems you are trying to load sample files and project files at once. Please select either a set of samples or a single project file to load into this session."